examples/hack-the-supergraph-ssr/app/error.tsx

```diff
@@ -4,16 +4,14 @@ import React from "react";
 import { Box, Heading, Text, VStack } from "@chakra-ui/react";
 
 export const Error = ({
-  children,
-  code,
   error,
-}: React.PropsWithChildren<{
-  code?: string;
-  error: { toString(): string };
-}>) => (
+  reset,
+}: {
+  error: Error;
```

Comment:
Updating our error pages to conform with the API as described in the latest docs: https://nextjs.org/docs/app/api-reference/file-conventions/error

Reply:
Interestingly, there's a lint error if I try to type `error` as `ApolloError` here:

![CleanShot 2023-06-14 at 11 46 38](https://github.com/apollographql/apollo-client-nextjs/assets/5139846/144a61f5-76f5-45f2-b877-5abc4a27103d)